src/components/modal/modal.ts

```diff
@@ -50,7 +50,7 @@ class ModalComponent implements WebComponent {
     document.querySelector('.modal')!.addEventListener('click', this.modalClick);
     document.querySelector('.modal-wrap')!.addEventListener('click', this.closeModal);
     if (this.modalContent.picture) {
-      document.querySelector('.modal__image')!.setAttribute('src', `src/assets/${this.modalContent.picture}.PNG`);
+      document.querySelector('.modal__image')!.setAttribute('src', `.assets/${this.modalContent.picture}.PNG`);
```

Comment:
document.querySelector('.modal__image')!.setAttribute('src', `./assets/${this.modalContent.picture}.PNG`);
tak powinno być najlepiej jak dla mnie, brakuję tam '/' pomiędzy kropką a 'assets'